metabox/metabox/core/scenario.py

```diff
@@ -305,3 +319,22 @@ def mktree(self, path, privileged=False, timeout=0, target="all"):
             cmd = ["sudo"] + cmd
         cmd_str = shlex.join(cmd)
         self.run_cmd(cmd_str, target=target, timeout=timeout)
+
+    def run_manage(self, args, privileged=False, timeout=0, target="all"):
+        """
+        Runs the manage.py script with some arguments
+        """
+        path = "/home/ubuntu/checkbox/metabox/metabox/metabox-provider"
+        cmd = f"bash -c 'cd {path} ; python3 manage.py {args}'"
```

Comment:
Possible candidate for a followup: lets stop making these commands via string concatenation. At least for args, this could be dangerous!

Reply:
I tried to avoid having everything in the same string and parse it as a list. The issue was that the `manage.py` script had to be executed from